fuse-products/src/main/java/software/tnb/product/integration/builder/AbstractIntegrationBuilder.java

```diff
@@ -465,4 +467,13 @@ public SELF useOcpCustomStrategy(OpenshiftCustomDeployer customStrategy) {
     public OpenshiftCustomDeployer getOCPCustomStrategy() {
         return customStrategy;
     }
+
+    public SELF runApplication(boolean runApplication) {
```

Comment:
is there a case where you want to set this anywhere else than GitIntegrationBuilder? if not, can it be moved there?

Reply:
yes it can be moved there, thanks